tests/checks/test_stablereq.py

```diff
@@ -91,6 +120,26 @@ def test_existing_stable_keywords(self):
         expected = StableRequest('0', ['~amd64'], 30, pkg=VersionedCPV('cat/pkg-2'))
         assert r == expected
 
+    def test_existing_stable_keywords_stable_time(self, stable_time):
```

Comment:
I would prefer to use some better format for the parameterized fixture:
```suggestion
    @pytest.mark.parametrize(("stable_days", "less_days", "more_days"), (
        pytest.param(None, (0, 1, 10, 20, 29), (30, 31), id="stable_time=unset"),
        pytest.param(1, (0,), (1, 10), id="stable_time=1"),
        pytest.param(14, (0, 1, 10, 13), (14, 15, 30), id="stable_time=14"),
        pytest.param(30, (0, 1, 10, 20, 29), (30, 31), id="stable_time=30"),
        pytest.param(100, (98, 99), (100, 101), id="stable_time=100"),
    ))
    def test_existing_stable_keywords_stable_time(self, stable_days, less_days, more_days):
```

That way you can remove all the previous fixture, and use direct arguments without polluting `self.` I also think it looks more clean as a fixture list.

Reply:
As I mentioned in https://github.com/pkgcore/pkgcheck/pull/429#issuecomment-1214202815, I thought I might have gone overboard. I'll parameterize the test

Reply:
Implemented in [16aeb72](https://github.com/pkgcore/pkgcheck/pull/429/commits/16aeb7281e82bb7ae0801f7589d0ce740c7ca44f)